gordon/interfaces.py

```diff
@@ -53,16 +53,29 @@ def update_phase(new_phase):
 
 
 class IGenericPlugin(Interface):
-    """**Do not** implement this interface directly.
+    """Plugin which is associated with a phase.
```

Comment:
Actually, you can remove the `phase` attribute here, it's not needed for runnable plugins.

Reply:
Yeah, this attribute is the main piece that made me split the cleaner out. I initially was thinking that that the event consumer would implement both runnable and message handler.

Reply:
I apologize for my short term memory since we _just_ talked about this yesterday 🤣 but does the `phase` attribute need to live here? It doesn't make sense to require a runnable to have a phase.

Reply:
Ah, good point. I just forgot to move it. Will fix!